ci_framework/roles/manage_secrets/README.md

```diff
@@ -0,0 +1,44 @@
+# manage_secrets
+Copy secret content such as pull-secret and ci-token to a (not really) secure
```

Comment:
what do you mean by (not really) secure

do you just mean 'known location' perhaps?

Reply:
if we were to use a secure location, that'd be HSM at this point.... But yeah, guess "known location" would do